lib/rrule/rule.rb

```diff
@@ -95,7 +95,8 @@ def all_until(start_date: nil, end_date: max_date, limit: nil)
     def parse_options(rule)
       options = { interval: 1, wkst: 1 }
 
-      params = rule.split(';')
+      # Remove RRULE: prefix to prevent parsing options incorrectly.
+      params = rule.delete_prefix('RRULE:').split(';')
```

Comment:
I was originally a conditional `delete_prefix!` to modify the string in place, but the library uses frozen string literals so it fails unless we disable it for the particular test. However after looking at the rest of the code, we don't really seem mutate things in-place, so this seems fine for now I think?